README.md

```diff
@@ -6,7 +6,7 @@ A set of Grafana dashboards and Prometheus alerts for OpenEBS that can be instal
 
 ## Status
 
-**Beta**. This repository currently supports dashboards and alerts for `cStor`, `Jiva`, `LVM LocalPV` OpenEBS storage engines.
+**Beta**. This repository currently supports dashboards and alerts for `Mayastor`, `LocalPV LVM`, `LocalPV ZFS` OpenEBS storage engines.
```

Comment:
Should this be 'Replicated PV Mayastor' instead?
cc: @avishnu

Reply:
Yes, @niladrih you are correct. Mayastor = "Replicated PV Mayastor"